app/assets/stylesheets/dm/_classes.scss

```diff
@@ -338,3 +338,19 @@ to confirm the class DOES NOT EXIST before adding a new class
 .maxw-none {
   @include u-maxw('none', !important);
 }
+
+.usa-nav__primary button[aria-expanded="false"], .usa-nav__primary button[aria-expanded="true"]{
```

Comment:
the chevron looks place slightly higher than in the figma -- maybe by a couple pixels (not sure if that's just me)

Reply:
I know.. Josh and I went back and forth on this.. a few times..  I did show this to Becca and she said it was good... but as you mentioned.. after this is merged back in to Master I'll deploy this to DEV and have Becca give it a final OK.